web/src/pages/index.js

```diff
@@ -119,12 +117,45 @@ export const query = graphql`
                 }
             }
         }
+        subHero: allSanityIndexPage {
+            edges {
+                node {
+                    subHero {
+                        heading
+                        info
+                    }
+                }
+            }
+        }
```

Comment:
You shouldn't need to use the `allSanityIndexPage` query for these. Try:
```
  sanityIndexPage {
    subHero {
      heading
      info
    }

    visionSection {
      heading
      info
    }
    ...
  }
```

This should make accessing the properties you need a little easier.

```
const { sanityIndexPage: { subHero, visionSection } } = data;

...

<BgImageWithCard
  data={subHero}
  ...
 >
  ```

Reply:
@asasmith ah okay this is so helpful! thank you!

Reply:
@kcbreuil yw. You'll only need to use the `all*` queries if you want to return a collection of pages (like blog posts).